StreamChatClient_v3/Controllers/ChannelController_Tests.swift

```diff
@@ -223,10 +223,18 @@ class ChannelController_Tests: StressTestCase {
         let channel: Channel = client.databaseContainer.viewContext.loadChannel(cid: channelId)!
         assert(channel.latestMessages.count == 1)
         let message: Message = channel.latestMessages.first!
+
+        // Send notification with event happened in the observed channel
+        let event = try MemberAddedEvent.dummy(cid: controller.channelQuery.cid, userId: .unique)
+        let notification = Notification(newEventReceived: event, sender: self)
+        client.webSocketClient.notificationCenter.post(notification)
```

Comment:
Let's create a separate test method for this, please. We will have many delegate methods, we need to start structuring it.

Reply:
fixed in 791b4a3673a2127949ad673e06f1ae90366330f9